pkg/geo/geofn.go

```diff
@@ -39,3 +41,75 @@ func STAreaGeography(g *Geography) (float64, error) {
 	}
 	return area, nil
 }
+
+// STContainsGeography exactly matches the semantics of ST_Contains.
+func STContainsGeography(g1, g2 *Geography) (bool, error) {
+	sf1, err := assertSingleFigure(g1.Figure)
+	if err != nil {
+		return false, err
+	}
+	sf2, err := assertSingleFigure(g2.Figure)
+	if err != nil {
+		return false, err
+	}
+	switch f1 := sf1.(type) {
+	case s2.Point:
+		switch f2 := sf2.(type) {
+		case s2.Point:
+			return f1.ContainsPoint(f2), nil
+		case *s2.Polygon:
+			// TODO(dan): Pretty sure this is wrong if polygon is of size 0.
```

Comment:
you can't have a 0 point polygon i believe. polygons have to be closed. see 4.3.5: https://postgis.net/docs/using_postgis_dbmanagement.html